vars/dockerExecute.groovy

```diff
@@ -1,53 +1,85 @@
 import com.cloudbees.groovy.cps.NonCPS
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.ContainerMap
+import com.sap.piper.JenkinsUtils
+import groovy.transform.Field
 
-def call(Map parameters = [:], body) {
+@Field def STEP_NAME = 'dockerExecute'
+@Field def PLUGIN_ID_DOCKER_WORKFLOW = 'docker-workflow'
 
-    def STEP_NAME = 'dockerExecute'
-    def PLUGIN_ID_DOCKER_WORKFLOW = 'docker-workflow'
+@Field Set GENERAL_CONFIG_KEYS = ['jenkinsKubernetes']
 
-    handlePipelineStepErrors(stepName: STEP_NAME, stepParameters: parameters){
-        def dockerImage = parameters.dockerImage ?: ''
-        Map dockerEnvVars = parameters.dockerEnvVars ?: [:]
-        def dockerOptions = parameters.dockerOptions ?: ''
-        Map dockerVolumeBind = parameters.dockerVolumeBind ?: [:]
```

Comment:
We loose some defaults here, is it ok to habe `null` instead of `[:]` and `''`?

Reply:
yes. All these parameters are checked by if statements before being used.